pkg/component/networking/vpn/shoot/shoot.go

```diff
@@ -667,6 +667,19 @@ func (v *vpnShoot) tunnelControllerContainer() *corev1.Container {
 				Add: []corev1.Capability{"NET_ADMIN"},
 			},
 		},
+		ReadinessProbe: &corev1.Probe{
```

Comment:
Should we also add a liveness probe when we are at it anyway? In other words, would it make sense to check the liveness of the container and restart it in case it is not healthy anymore?